edb/edgeql/compiler/__init__.py

```diff
@@ -201,6 +202,7 @@ def compile_ast_to_ir(
     *,
     script_info: Optional[irast.ScriptInfo] = None,
     options: Optional[CompilerOptions] = None,
+    inlining_context: Optional[context.ContextLevel] = None,
```

Comment:
You don't use these entrypoints anymore, right? Can we drop the `inlining_context` from them?